tools/mtouch/mtouch.cs

```diff
@@ -629,7 +589,7 @@ public static string GenerateMain (Target target, IEnumerable<Assembly> assembli
 					sw.WriteLine ("\txamarin_init_mono_debug = {0};", app.PackageManagedDebugSymbols ? "TRUE" : "FALSE");
 					sw.WriteLine ("\txamarin_executable_name = \"{0}\";", assembly_name);
 					sw.WriteLine ("\tmono_use_llvm = {0};", enable_llvm ? "TRUE" : "FALSE");
-					sw.WriteLine ("\txamarin_log_level = {0};", verbose.ToString (CultureInfo.InvariantCulture));
+					sw.WriteLine ("\txamarin_log_level = {0};", Verbosity.ToString (CultureInfo.InvariantCulture));
```

Comment:
since it's an `int` there's not point in supplying a `CultureInfo` here

Reply:
my bad, it's fine - current culture could insert a space (for thousands), should not happen but...